apps/builder/app/builder/features/settings-panel/props-section/props-section.tsx

```diff
@@ -207,6 +209,20 @@ export const PropsSection = (props: PropsSectionProps) => {
   );
 };
 
+const getPropTypeAndValue = (value: unknown) => {
```

Comment:
if value is always just the value, you could just have a function that: 
```js
const type = inferValueType(value)
const smth = {type, value}
```

Reply:
need a pair to pass to union